app/main.py

```diff
@@ -11,6 +13,10 @@
 from sdc.rabbit.publisher import QueuePublisher
 from sdc.rabbit.exceptions import QuarantinableError, RetryableError
 from sdc.rabbit.consumers import MessageConsumer
+from tornado.httpclient import AsyncHTTPClient, HTTPError
+import tornado.httpserver
+import tornado.ioloop
+import tornado.web
```

Comment:
Why have these changed if it's just logging changes?

Reply:
Think it's just the ordering

Reply:
Yea, it's just the ordering

Reply:
Fair enough